src/app/page.tsx

```
@@ -1,7 +1,110 @@
+        value={password} 
+        onChange={(e)=> setPassword(e.target.value)}
+        />
+
+        <Button onClick={onSubmit}>
+          create account
+        </Button>
+
+        </div>
+
+
+        //
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)